apps/docs/content/guides/platform/manage-your-usage/storage-size.mdx

```diff
@@ -0,0 +1,74 @@
+---
+id: 'manage-usage-storage-size'
+title: 'Manage Storage size usage'
+---
+
+## What you are charged for
+You are charged for the total size of all assets in your buckets.
+
+## How charges are calculated
+Storage size is charged by Gigabyte-Hours (GB-Hrs). 1 GB-Hrs represents the use of 1 GB of storage for 1 hour.
+For example, storing 10 GB of data for 5 hours results in 50 GB-Hrs (10 GB × 5 hours).
+
+### Storage size usage on your invoice
+Storage size usage is shown as "Storage Size GB-Hrs" on your invoice.
+
+## Pricing
+1 GB-Hrs is $0.00002919. That is $0.021 per GB per month. You are only charged for usage exceeding your subscription plan's free quota.
```

Comment:
Kept using "free quota" for now. See https://github.com/supabase/supabase/pull/33275#discussion_r1939924751